beacon-chain/powchain/service.go

```diff
@@ -278,6 +287,7 @@ func (w *Web3Service) ProcessChainStartLog(VRClog gethTypes.Log) {
 	log.WithFields(logrus.Fields{
 		"ChainStartTime": chainStartTime,
 	}).Info("Minimum Number of Validators Reached for beacon-chain to start")
+	w.chainStartFeed.Send(chainStartTime)
```

Comment:
Where is the test assertion that this happens? This is now critical logic to running the beacon chain node and must have an assertion